module/product/src/Application/Controller/Api/ProductCreateAction.php

```diff
@@ -102,8 +103,9 @@ public function __construct(
     public function __invoke(Request $request): Response
     {
         $type = $request->request->get('type', SimpleProduct::TYPE);
```

Comment:
Left default value temporarily for frontend compatibility. Once resolved default value should be removed